app/policies/message_policy.rb

```diff
@@ -5,7 +5,7 @@ def create?
     elsif developer_sender?
       true
     elsif business_sender?
-      permission.can_message_developer?(role_type: developer.role_type)
+      permissions.can_message_developer?(role_type: developer.role_type)
```

Comment:
```suggestion
      user.permissions.can_message_developer?(role_type: developer.role_type)
```

And then we can remove the private helper method, too.